generate_changelog.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# default arguments
+host="https://github.com"
+api_host="https://api.github.com"
+origin="sendgrid"
+repo="sendgrid-python"
+authorization_token=""
+
+# helpers
+usage() {
+  cat << EOF
+Usage:
+  generate_changelog [-h]
+  generate_changelog [-t]
+  generate_changelog [-v]
+
+Options:
+  -h        Print usage information
+  -t        Specify a token to authenticate with the api that gets PR titles
+            May also set the AUTHORIZATION_TOKEN environment variable
+  -v        Specify the new version number
+EOF
+}
+
+create_changelog() {
+
+  content=""
+  # Get merge commits
+  merges="$(git log ${latest_release}..HEAD --merges --format="* %s (%h)" --grep="^Merge pull request #" | cut -d "#" -f2 | cut -d " " -f1)"
+  for pr in $merges; do
+    echo "Getting title of PR #$pr"
+    url="$host/$origin/$repo/pull/$pr"
+    api_url="$api_host/repos/$origin/$repo/pulls/$pr"
+    curl_command="curl -s $api_url"
+    if [ "$authorization_token" != "" ]
+      then
+        curl_command="curl -H \"Authorization: token $authorization_token\" -s $api_url"
```

Comment:
I would suggest:
`curl_command="$curl_command -H \"Authorization: token $authorization_token\""`
So there is a single place to update the curl command if needed in the future.